modules/core/src/main/scala/fs2/kafka/ProducerSettings.scala

```diff
@@ -224,7 +241,8 @@ object ProducerSettings {
     override val valueSerializer: Resource[F, ValueSerializer[F, V]],
     override val customBlockingContext: Option[ExecutionContext],
     override val properties: Map[String, String],
-    override val closeTimeout: FiniteDuration
+    override val closeTimeout: FiniteDuration,
+    override val failFastProduce: Boolean = false
```

Comment:
Instead of defining here the default value, wouldn't it be more convenient to set it in the `create` method where the other default values are set?

Reply:
It's addressed in commit: 5472f259d830cb6c411d8e286bc9a76c4238cd40